docs/JestObjectAPI.md

```diff
@@ -312,6 +313,21 @@ test('works too', () => {
 
 Returns the `jest` object for chaining.
 
+### `jest.retryTimes()`
```

Comment:
should mention it just works with circus

Reply:
I mention in that section that it works with jest-circus. Should I be more explicit?

Reply:
Where is it mentioned?

Reply:
Ah my mistake sorry. I mentioned it in the changelog not the docs. Will update.